ext/reflection/tests/gh15902/ReflectionClass-callable.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+ReflectionClass object default property - used to say "callable"
+--FILE--
+<?php
+
+class C {
+	public stdClass $a = FOO;
+}
+define('FOO', new stdClass);
+
+new C;
+
+$reflector = new ReflectionClass(C::class);
+var_dump( (string)$reflector );
```

Comment:
Please remove the spaces inside parens

Reply:
Also, this can just use `echo $reflector;`, the `var_dump` doesn't really have a purpose.